### DISCUSSION TITLE
Modifying Booleans

### DISCUSSION THREAD

**Discussion by Zebratic** [jbevain/cecil#717] [Q&A] [answered]
I am using Mono.Cecil for C# .NET Framwork 4.7.2, i have an assembly with a settings file (settings.cs)
In there i have a few bools and strings that i want to modify from another exe (exe builder).
How do i load the assembly, modifying the bools and strings, and then outputting the modified assembly to an exe file?

**Comment by jbevain** [comment, marked as answer]
I suggest you have a look at the Cecil tests, there's everything in there showing how to read/modify and write back an assembly.
